app/common/containers/forms/DateFilterField/index.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import withStyles from "withStyles";
+import { Field } from "redux-form";
+
+import FieldDate from "components/reduxForm/FieldDatepicker";
+
+import styles from "./styles.scss";
+
+@withStyles(styles)
+export default class DateFilterField extends Component {
+  componentDidMount() {
+    this.initForm();
+  }
+
+  render() {
+    const { filters } = this.props;
+
+    return (
+      <div className={styles.date}>
+        {filters.map(({ name, title, placeholder }) => (
+          <div key={name} className={styles.date__filter}>
+            <Field
+              name={name}
+              component={FieldDate}
+              dateFormat="YYYY-MM-DD"
+              labelText={title}
+              placeholder={placeholder}
+            />
+          </div>
+        ))}
+      </div>
+    );
+  }
+
+  initForm() {
```

Comment:
You mean `initField` ? Because I cant find form in this component.

Reply:
Nope. Because we are changing the `initialValues` of the whole form here